src/App/Components/ArticleNotFound/ArticleNotFound.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Navbar from '../Common/NavProfile/navbar';
+
+const ArticleNotFound = () => (
+  <React.Fragment>
+    <Navbar />
+    <div className="nFound">
```

Comment:
Work on this class name this is CSS?
http://bdavidxyz.com/blog/how-to-name-css-classes/

Reply:
On it